src/MusicStore/config.json

```diff
@@ -1,4 +1,7 @@
 ﻿{
+    "ApplicationInsights": {
+        "InstrumentationKey": "797d1481-9988-48f2-b41b-f4e305cd9207"
+    },
     "DefaultAdminUsername": "Administrator@test.com",
```

Comment:
Are you referring to an Azure subscription to view the Application Insights? I don't think we can enable public access to a particular Application Insights resource.

What do you think about leaving the InstrumentationKey blank with a comment explaining how to add it? This will silently disable sending of telemetry to Application Insights and allow each user to use their own subscription to configure and see telemetry.